frontend/src/components/input.rs

```diff
@@ -0,0 +1,36 @@
+use yew::prelude::*;
+
+#[derive(Clone, PartialEq, Properties)]
+pub struct InputFieldProps {
+    // pub input_value: String,
+    // pub on_cautious_change: Callback<ChangeData>,
```

Comment:
```suggestion
```
?

Reply:
why are these commented out?

Reply:
They are causing some weird errors in <Input Field> components and I havent figured out the implementation of on_cautious_change yet.